wallet/src/repository/wallet/mod.rs

```diff
@@ -313,23 +319,22 @@ where
         // Query database `transaction_next_id` to compute total amount of transactions
         let db_total = self
             .db
-            .get_or_default::<_, u32>(&keys::transaction_next_id(account))?
-            .saturating_sub(1);
+            .get_or_default::<_, u32>(&keys::transaction_next_id(account))?;
```

Comment:
Why remove the `.saturating_sub(1)`?

Reply:
It was by the error in the indexes of the wallet

Reply:
Then can we add tests so it doesn't happen again? :P

Reply:
![image](https://user-images.githubusercontent.com/12464544/94420860-0ae52680-0185-11eb-8854-1f0448bf99c3.png)

:rofl:

Reply:
Fine, I'll do it myself

https://github.com/tmpolaczyk/witnet-rust/tree/wallet-get-transactions-tests